week2/homework/src/index.js

```diff
@@ -1,3 +1,97 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+const fileName = `${__dirname}/todoList.txt`;
+
+const command = process.argv[2];
+const args = process.argv.slice(3);
+
+const newItem = process.argv.slice(4);
+
+const number = parseInt(args);
```

Comment:
`args` is an array, so you should select an item of this array to parse it as int, but because JavaScript is merciful, it still works, the first element of the array will be converted to int. Try not to rely on JavaScript flexibility and write more comprehensive code.